README.rst

```diff
@@ -0,0 +1,70 @@
+pyincore-viz
+============
+
+**pyincore-viz** is a Python project that provides visualization and other utilities for use with **pyincore**
+The development is a part of NIST sponsored IN-CORE (Interdependent Networked Community Resilience Modeling
+Environment) initiative. 
+
+Mac specific notes
+------------------
+
+- We use `matplotlib` library to create graphs. There is a Mac specific installation issue addressed at `here1 <https://stackoverflow.com/questions/4130355/python-matplotlib-framework-under-macosx>`_ and 
```

Comment:
should here1 and here2 just be "here" and "here" or was it intentional to name them separately as here1 and here2?

Reply:
rst doesn't like here and here. It says that it has duplicated reference

Reply:
@jonglee1 can you update version in setup.py as well?